Project-II-Calculator/calculator/src/App.css

```diff
@@ -0,0 +1,15 @@
+/* ___   _   _    ___ _   _ _      _ _____ ___  ___ 
```

Comment:
As crazy as it looks, I like the fact that you decided to comment what this styling was for. I don't recommend you spend much time on this though, a simple comment should do just fine. You don't want employers thinking you spend too much time on stuff like this.